src/index.ts

```diff
@@ -128,6 +129,7 @@ export interface IDKeys {
 }
 
 export interface CreateOptions {
+  lifetimeNs?: bigint
```

Comment:
Pretty sure it has to be added here to be allowed as a valid setting to override?  Can you clarify where you see a lifetime already exposed from the create function?

Reply:
https://github.com/ipfs/js-ipns/blob/16e0e10682fa9a663e0bb493a44d3e99a5200944/src/index.ts#L163-L166

See the `lifetime` argument.

Either way, I think we can just call it `ttlNs` rather than lifetime, since lifetime is just a UX proxy for the expiration which has a different role to the lifetime. The fact that the ttl was already derived from the lifetime/expiration was unnecessarily confusing.